string Request
---
string TimeNow
string DateNow
duration timestamp